newm-server/src/main/kotlin/io/newm/server/features/earnings/repo/EarningsRepositoryImpl.kt

```diff
@@ -46,11 +38,14 @@ import org.quartz.JobBuilder.newJob
 import org.quartz.JobKey
 import org.quartz.SimpleScheduleBuilder.simpleSchedule
 import org.quartz.TriggerBuilder.newTrigger
+import java.math.BigDecimal
+import java.time.Instant
+import java.time.LocalDateTime
+import java.util.*
 
 private const val CLAIM_ORDER_INTER_PROCESS_MUTEX_PATH = "/mutexes/claim-order"
 
 class EarningsRepositoryImpl(
-    private val userRepository: UserRepository,
```

Comment:
I had to eliminate UserRepo here because it now causes circular dependencies :UserRepo -> ReferralHeroRepo -> EarningsRepo -> UserRepo